README.md

```diff
@@ -4,7 +4,9 @@
 
 [dox]: https://docs.rs/ctest2
 
-**Note: This is a fork repository and we intend to use this in libc-test only.**
+**Note: This is a fork of [`ctest`], intended as a temporary replacement until maintenance of [`ctest`] resumes or ownership is transferred.**
```

Comment:
```suggestion
**Note: This is a fork of [`ctest`], intended as a temporary replacement until maintenance of [`ctest`] resumes.**
```

I don't expect the latter.

Reply:
Also, this doesn't mention the use in libc.

Reply:
Ok, force-pushed suggested amendment.